.github/workflows/cherry-pick.yml

```diff
@@ -0,0 +1,55 @@
+name: Cherry-pick a release
+
+on:
+  workflow_dispatch:
+    inputs:
+      amp-version:
+        description: AMP version (13 digits)
+        required: true
+        type: string
+      sha-1:
+        description: commit sha to cherry-pick with
+        required: true
+        type: string
+      sha-2:
+        description: (optional) commit sha to cherry-pick with
+        required: false
+        type: string
+#      TODO(estherkim): tie to promote workflow
+#       channel:
+#         description: channels to promote to
+#         required: true
+#         type: choice
+#         options:
+#           - nightly
+#           - traffic
+#           - beta
+#           - stable
+#           - lts
+
+jobs:
+  cherry-pick:
+    runs-on: ubuntu-latest
+
+    environment: cherry-pick
+
+    steps:
+      - name: Checkout ampproject/amphtml
```

Comment:
I think it makes more sense to have this workflow in the `ampproject/amphtml` repo, instead of here - it feels a bit tacked-on here, wdyt?

Reply:
Wouldn't it be better UX for on-duty engineers to come to a single place - cdn-configuration - for all release duties?

Reply:
The thing is, I think it would confuse the on-duty that you can change repo X from repo Y, I personally would have expected the cherry-pick action to be in the repo that gets cherry-picked. We could do a quick show-of-hands poll with the release-on-duty people to see what they think?

Reply:
Can we start by leaving this here? Ok to move if it gets confusing but I think having cdn-configuration actions as the new dashboard makes a lot of sense.

Reply:
Haha sure, let's leave it here for now. I would still insist we ask the question though, better to get feedback about this asap